doc/manual/installation/installing-binary.xml

```diff
@@ -136,6 +136,110 @@ sudo rm /Library/LaunchDaemons/org.nixos.nix-daemon.plist
 
 </section>
 
+<section xml:id="sect-apfs-volume-installation">
+  <title>APFS Volume Installation</title>
+
+  <para>
+    The root filesystem is read-only as of macOS 10.15 Catalina, all writable
+    paths to a separate data volume.  This means creating or writing to <filename>/nix</filename>
+    is not allowed.  While changing the default prefix would be possible, it's
+    a very intrusive change that has side effects we want to avoid for now.
+  </para>
+
+  <para>
+    For common writable locations <literal>firmlinks</literal> where introduced,
+    described by Apple as a "bi-directional wormhole" between two filesystems.
+    Essentially a bind mount for APFS volumes.  However this is (currently) not
+    user configurable and only available for paths like <filename>/Users</filename>.
```

Comment:
but it is user configurable via synthetic.conf?

Reply:
"it is only configurable for paths like /Users/" -- not specific. Why even state this when next paragraph says the same thing but is more specific.

Reply:
no synthetic.conf doesn't create firmlinks, with an actual firmlink it wouldn't necessary to create a separate volume and all the configuration that comes with it.